bedrock/base/templates/macros.html

```diff
@@ -82,27 +79,18 @@ <h2 class="mzp-u-title-xs thank-you hidden">{{ ftl('send-to-device-your-download
             <input type="hidden" name="product" value="{{ product }}">
           </div>
           <div class="mzp-c-field mzp-l-stretch">
-            <label class="mzp-c-field-label" for="{{ id }}-input" data-alt="{% if input_label_alt %}{{ input_label_alt }}{% else %}{{ ftl('send-to-device-enter-your-email-or-phone', fallback='send-to-device-enter-your-email-or-phone-10-digit') }}{% endif %}">
+            <label class="mzp-c-field-label" for="{{ id }}-input">
               {% if input_label %}
                 {{ input_label }}
               {% else %}
                 {{ ftl('send-to-device-enter-your-email') }}
               {% endif %}
             </label>
-            <input id="{{ id }}-input" class="mzp-c-field-control send-to-device-input" name="phone-or-email" type="text" required>
+            <input id="{{ id }}-input" class="mzp-c-field-control send-to-device-input" name="s2d-email" type="text" required>
```

Comment:
Might renaming this break something on the receiving end?

Reply:
I think the change in `bedrock/firefox/views.py` is what was needed to avoid problems. Unless @pmac says different.

Reply:
Seems ok to me, I tested the form end-to-end and got the email ok 👍